Bottom: 4fb9f561583eb3c9df5b4220175ac6e7f40e6382
Top:    d14a6f1ab029d7ad1d4bea68b98641cc2a941760
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 11:38:49 +0000

thermal.c: Add #warning pointing out where we might get a speedup




---

diff --git a/src/thermal.c b/src/thermal.c
index 445b97d..fa87f28 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -432,6 +432,7 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
         arc = CirclePoly (pin->X, pin->Y, pin->Thickness / 2);
         /* create a thin ring */
         poly_Boolean_free (pa, arc, &m, PBO_SUB);
+#warning THIS WOULD BE FASTER IF WE JUST SUTRACTED A CROSS SHAPE
         /* fix me needs error checking */
         if (style == 2)
           {
